src/ReportedBySelect/index.js

```diff
@@ -168,7 +168,7 @@ const ReportedBySelect = ({
 
     if (isMulti) {
       if (value.length) {
-        return value.map((item) => item.hidden
+        return value.map((item) => item?.hidden
```

Comment:
Some reason the lead array was having 1 item being undefined, not sure how this happened but it was blocking me, I'm still trying to reproduce the issue

Anyways, that is not related with this ticket, but when I get something more reproducible I'll come back here and check if this was indeed the fix for it

Reply:
I reset this change, instead I added the [validation here](https://github.com/PADAS/das-web-react/pull/1170/files#r1787908920)